src/components/AreaChart.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { omit } from 'lodash';
+import { Subtract } from 'utility-types';
+import { VisualizationObject, AFM } from '@gooddata/typings';
+
+import { AreaChart as AfmAreaChart } from './afm/AreaChart';
+
+import { ICommonChartProps } from './core/base/BaseChart';
+import { convertBucketsToAFM } from '../helpers/conversion';
+import { generateStackedDimensions } from '../helpers/dimensions';
+import { isStackedChart } from '../helpers/stacks';
+
+export interface IAreaChartBucketProps extends ICommonChartProps {
+    measures: VisualizationObject.BucketItem[];
+    attributes?: VisualizationObject.IVisualizationAttribute[];
```

Comment:
In mean time names of props for individual buckets have changed to form of eg. `measures`, `trendBy`, `segmentBy`. Pls synchronize names in Area chart to this form and fix it also in stories and examples